src/interface/src/app/account-dialog/account-dialog.component.ts

```diff
@@ -133,7 +137,13 @@ export class AccountDialogComponent implements OnInit {
           });
         },
         (err) => {
-          this.error = err;
+          if (err.status == 401) {
+            this.error = {
+              message: 'Your credentials were not entered correctly.',
+            };
+          } else {
+            this.error = { message: 'An unexpected error occured.' };
```

Comment:
I don't love this, but I do think we need a catch-all for exceptions somehow, especially since we're not making consistent use of error codes. Open to suggestions.

Reply:
only suggestion for this is that we might want to put this in a constant somewhere if its going to be the "generic" one

Reply:
I agree. I was also thinking it would be nice to add a ticket to standardize these strings, as well as tidy up some of the backend response codes.